src/models/Factor.js

```diff
@@ -212,7 +212,11 @@ function (Okta, Q, factorUtil, Util, Errors, BaseLoginModel) {
         return {'password': Okta.loc('error.password.required')};
       }
     },
-
+    addPasscode: function () {
+      // for new idx pipeline with email magic link we dont need to show otp code input field,
```

Comment:
nit: I think `// we don't need passcode in email with magic link` should be enough